### DISCUSSION TITLE
我想让gpt自己生成代码，然后把代码插入我的环境里执行并获取返回结果要怎么实现

### DISCUSSION THREAD

**Discussion by dao-zero** [lobehub/lobehub#1262] [Development Question]
如题，我想写一个插件，这个插件让gpt先根据问题写出相应的解决代码，然后把代码发送到我的插件执行，然后接收返回值，要怎么操作呢，我需要怎么修改：
![image](https://github.com/lobehub/lobe-chat/assets/83216969/1ce9210a-0d1d-4947-8b22-c9c8b7e45812)

**Comment by buwanyuanshen** [comment]
你需要白日做梦😓，需要什么去百度，这里都是不会的。
